build.gradle

```diff
@@ -538,6 +538,8 @@ project(':core') {
     duplicatesStrategy 'exclude'
   }
 
+  systemTestLibs.dependsOn('jar', 'copyDependantTestLibs')
```

Comment:
@ijuma Sorry I missed that. Thank you for pointing it out. Have combined the two into one block.